test/e2e/features/basic.feature

```diff
@@ -49,12 +51,17 @@ Feature: Basic test
         And application "dev-env" should be in "Synced" state
         And application "stage-env" should be in "Synced" state
         And application "cicd-app" should be in "Synced" state
-        Then executing "oc delete -k config/argocd"
-        Then executing "cd .." succeeds
+        When executing "kam webhook create --git-host-access-token $GIT_ACCESS_TOKEN --env-name dev --service-name taxi" succeeds
+        Then stderr should be empty
+        Then executing "kam webhook delete --git-host-access-token $GIT_ACCESS_TOKEN --env-name dev --service-name taxi" succeeds
```

Comment:
Should we also include a scenario for adding a webhook to the gitops repository?

Reply:
Will create a separate pr for it